Test/CoreSDK.Test/Net40/TaskExTests.cs

```diff
@@ -0,0 +1,173 @@
+namespace Microsoft.ApplicationInsights
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Assert = Xunit.Assert;
+
+    [TestClass]
```

Comment:
Documentation comments and general StyleCop compliance would be really appreciated.  Thank-you. :)